src/party_context_menu.c

```diff
@@ -129,31 +131,22 @@ static const u8 sButtonRects[][4] = {
 };
 
 static const s8 sButtonWindowIDs[][2][8] = {
-    {
-        { 0, -1, -1, -1, -1, -1, -1, -1 },
-        { 7, -1, -1, -1, -1, -1, -1, -1 }
-    }, {
-        { 0, 7, -1, -1, -1, -1, -1, -1 },
-        { 8, 7, -1, -1, -1, -1, -1, -1 }
-    }, {
-        { 0, 1, 7, -1, -1, -1, -1, -1 },
-        { 8, 9, 7, -1, -1, -1, -1, -1 }
-    }, {
-        { 0, 1, 2, 7, -1, -1, -1, -1 },
-        { 8, 9, 10, 7, -1, -1, -1, -1 }
-    }, {
-        { 0, 1, 2, 7, 3, -1, -1, -1 },
-        { 8, 9, 10, 11, 7, -1, -1, -1 }
-    }, {
-        { 0, 1, 2, 7, 3, 4, -1, -1 },
-        { 0, 1, 2, 3, 4, 8, -1, -1 }
-    }, {
-        { 0, 1, 2, 7, 3, 4, 5, -1 },
-        { 0, 1, 2, 3, 4, 5, 8, -1 }
-    }, {
-        { 0, 1, 2, 7, 3, 4, 5, 6 },
-        { 0, 1, 2, 7, 3, 4, 5, 6 }
-    },
+    { { 0, -1, -1, -1, -1, -1, -1, -1 },
+        { 7, -1, -1, -1, -1, -1, -1, -1 } },
```

Comment:
This might improve things, maybe
```suggestion
    { { 0, -1, -1, -1, -1, -1, -1, -1, },
        { 7, -1, -1, -1, -1, -1, -1, -1, } },
```